plane/source/docs/quadplane-frame-setup.rst

```diff
@@ -41,53 +40,59 @@ layout. For Tri and Y6 this parameter is ignored.
+-  **Output 7:** Motor 3 - Rear left top motor, counter-clockwise
+-  **Output 8:** Motor 4 - Rear right top motor, clockwise
+-  **Output 9:** Motor 5 - Front left bottom motor, counter-clockwise
+-  **Output 10:** Motor 6 - Front right bottom motor, clockwise
+-  **Output 11:** Motor 7 - Rear right bottom motor, counter-clockwise
+-  **Output 12:** Motor 8 - Rear left bottom motor, clockwise
 
 You can remember the clockwise/counter-clockwise rule for an octa-quad
 by "top motors turn in towards the fuselage, bottom motors turn out
 away from the fuselage".
 
+For a Tricopter configuration, the default output assignment is:
+
+-  **Output 5:** Motor 1 - Front right motor, looking from above
+-  **Output 6:** Motor 2 - Front left motor
+-  **Output 8:** Motor 4 - Rear motor
+-  **Output 11:** Motor 7 - Tail Tilt Servo (see below)
+
 The normal plane outputs are assumed to be on 1 to 4 as usual. Only
-vertical lift outputs (5 to 8 on a quad setup) run at high PWM rate
-(400Hz). In a quad setup you can also use channels 9 to 14 in any way
-you like, just as with the normal Plane code.
+vertical lift outputs (5 to 8 on a quad setup) normally should be run at high PWM rate
+(400Hz). When using these default configurations, you can assigne other outputs to whatever functions you desire.
 
-You can optionally move the quad motors to be on any other channel above
-4, using the procedure outlined in the section further below.
+You can optionally move the motors to be on any other channel, using the procedure outlined in the section further below.
 
 Tricopter
 ---------
 
-Frame Type 7 is Tricopter and can be either non-Tiltrotor configuration, or :ref:`Tiltrotor<guide-tilt-rotor>` configured using either Vectored or Non-Vectored yaw control. If using non-Tiltrotor or Non-Vectored Yaw Tilt-rotor, the yaw control output is setup as Motor 7 (``SERVOn_FUNCTION`` = 39) using a tilt mechanism for the yaw motor, Motor 4. You should set up the yaw servo’s maximum lean angle in degrees with :ref:`Q_M_YAW_SV_ANGLE<Q_M_YAW_SV_ANGLE>`. This lean angle assumes that ``SERVOn_MIN`` and ``SERVOn_MAX``, represent +/- 90 degrees, with ``SERVOn_TRIM`` representing 0 degrees lean.
+Frame Type 7 is Tricopter and can be either non-Tiltrotor, or a :ref:`Tiltrotor<guide-tilt-rotor>` configuration for the front motors, but using a separate servo to sideways tilt the rear motor for yaw control, or a :ref:`Tiltrotor<guide-tilt-rotor>` using Vectored Yaw control via the front tilting motors.
 
+If using a tilting yaw control, an output is setup as Motor 7 (``SERVOn_FUNCTION`` = 39) for a servo-controlled sideways tilt mechanism for the yaw motor, Motor 4. You should set up the yaw servo’s maximum lean angle in degrees with :ref:`Q_M_YAW_SV_ANGLE<Q_M_YAW_SV_ANGLE>` to prevent prop strikes to the ground or frame. This lean angle assumes that ``SERVOn_MIN`` and ``SERVOn_MAX``, represent +/- 90 degrees, with ``SERVOn_TRIM`` representing 0 degrees lean.
```

Comment:
I think we need to mention that the Tail Tilt servo is ignored if using Vectored Yaw. I think this is important because it's confusing that this output gets created, so important to note that even if it appears in the outputs it will not be used in this case.

Reply:
I will have to check, but I do not think that the servo is disabled....in other words, you could conceivably use both yaw methods I think at the same time....

Reply:
tilt mask has nothing to do with the yaw tilt servo....its really not a tilt motor....tilt motors become fixed wing propulsion motors...the tricopter yaw servo tilt is not involved

Reply:
It might be good to say exactly that. If I find it confusing I'm sure others will.

Reply:
doen